Update metric-alerts.mdx (#2857)
getsentry/sentry-docs: src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx

## 3: Choose a Time Window

> Choose the time period that you want to have your metric evaluated over. Your choices range between one minute and one day.

Reviewer: ```suggestion
Choose the time period over which to evaluate your metric. Your choices range between one minute and one day.
```

Reviewer: Minor edit above to catch the idea of aggregating.

<Note>

> We evaluate this time window every minute. As an example, with an hour time window, we'd evaluate time periods:

Reviewer: ```suggestion
Sentry evaluates the specified window each minute. For example, if you specify an hour time window, Sentry evaluates:

```
